src/string-utils.ts

```diff
@@ -1,9 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 export class StringUtils {
-    public static isNotEmpty(str: string): boolean {
+    public static isNotEmpty(str?: string): boolean {
```

Comment:
I think we generally prefer exporting functions directly, rather than exporting a class that exclusively contains static methods